TacticianBundle.php

```diff
@@ -11,8 +10,10 @@ class TacticianBundle extends Bundle
     public function build(ContainerBuilder $container)
     {
         parent::build($container);
-        $container->addCompilerPass(new DoctrineMiddlewarePass());
-        $container->addCompilerPass(new CommandHandlerPass());
+        $container->addCompilerPass(new Compiler\DoctrineMiddlewarePass());
```

Comment:
Ok ! Both of them require a suggested dependency as we want the bundle could be used without these middlewares.

Moving to the compiler pass prevents to declare these services that will just not working. Currently we get exception at runtime when trying to use middleware with uninstalled deps. Now we get error at compile time.